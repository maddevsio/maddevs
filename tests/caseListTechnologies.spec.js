import { mount } from '@vue/test-utils'
import ListTechnologies from '@/components/Cases/ListTechnologies'

describe('list technologies', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(ListTechnologies, {
      slots: {
        default: '<li>List item</li>',
      },
    })
  })

  // ------ IMPORTANT ----- //
  it('is a Vue instance', () => {
    expect(wrapper.exists()).toBeTruthy()
  })

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
  // --------------------- //

  it('should have correct data in slot', () => {
    expect(wrapper.html()).toContain('<li>List item</li>')
  })
})
