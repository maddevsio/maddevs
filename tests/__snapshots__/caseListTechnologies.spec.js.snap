// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`list technologies renders correctly 1`] = `
<ul
  class="case_list-technologies"
>
  <li>
    List item
  </li>
</ul>
`;
